src/main/java/nisum/marketplace/backend/service/CartItemsServiceImpl.java

```diff
@@ -0,0 +1,68 @@
+package nisum.marketplace.backend.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import nisum.marketplace.backend.model.CartItems;
+import nisum.marketplace.backend.repository.CartItemsRepo;
+
+@Service
+public class CartItemsServiceImpl implements CartItemsService {
+
+	@Autowired
+	CartItemsRepo cartRepo;
+	
+	/*
+	 * handle update by just creating an item if it doesnt exit
+	 * 
+	 * */
+	@Override
+	public CartItems updateCart(CartItems cartItems) {
+		return cartRepo.save(cartItems);
```

Comment:
I believe you should check if the cart exists first before updating it in case a bad request is passed

Reply:
I handle update by just overwriting whatever in the database with the pass in CartItems. If I want to change just one thing, i will use a patch endpoint